react/features/participants-pane/components/web/ParticipantItem.js

```diff
@@ -146,7 +146,6 @@ function ParticipantItem({
         <ParticipantContainer
             id = { `participant-item-${participantID}` }
             isHighlighted = { isHighlighted }
-            local = { local }
```

Comment:
was this removed intentonally? I see that it's still being used by the styled component

Reply:
Added it back